documentation/source/newsfragments/2434.bugfix.rst

```diff
@@ -0,0 +1 @@
+Selecting tests from a module that is not the first on the :make:var:`MODULE` list will now work correctly.
```

Comment:
I would think this would be a "change". And the text should read
```suggestion
Selecting tests using :make:var:`TESTCASE` will now search for the first occurrence of a test of that name in order of :make:var:`MODULE`s, and not just the first module.
```

Reply:
Changed in 0e7e5da8b21482cffeca7f66299555edf20a1c00.